twittermap/public/javascripts/common/services.js

```diff
@@ -1,6 +1,7 @@
 angular.module('cloudberry.common', [])
   .factory('cloudberryConfig', function(){
     return {
+      ws: config.wsURL,
```

Comment:
exactly. I'm mainly want to pass the `cloudberry.ws` (which specified in `application.conf` of `twittermap`) to Angular. do you have other elegant ways of doing it?